### ISSUE TITLE
Paroxetine Nerf

### ISSUE THREAD

**Issue by Jennalele** [Aurorastation/Aurora.3#862]
Paroxetine is a chemical that allegedly has a ten percent chance of causing the message "Your mind breaks apart..." followed by a bout of strength-200 hallucinations. 

Except the chance honestly feels more like 50%. I'm not sure if this is a bug or I'm just REALLY unlucky, but it feels like it really needs to be lowered further. Most shifts if I'm playing my character that uses paroxetine, I can now expect to trip balls at LEAST 4 times in a 2 hour shift. 

TL;DR: Paroxetine does more harm than help right now, please, the angsty snowflakes are tripping mad balls.

**Comment by NanakoAC** [comment]
IIRC its a 10% chance which procs every ~50 seconds, so that does add up to a high likelihood of it ocurring over a round

**Comment by NanakoAC** [comment]
fixed by #1897
